olm/prepare-community-operators-update.sh

```diff
@@ -132,39 +134,32 @@ do
     echo $platform
     echo "     => will create release '${lastPackagePreReleaseVersion}' in the following package folder :'${folderToUpdate}'"
 
-    mkdir -p "${folderToUpdate}/${lastPackagePreReleaseVersion}"
+    mkdir -p "${folderToUpdate}/${lastPackagePreReleaseVersion}/manifests"
+    mkdir -p "${folderToUpdate}/${lastPackagePreReleaseVersion}/metadata"
     sed \
     -e "/^  replaces: ${packageName}.v.*/d" \
     -e "/^  version: ${lastPackagePreReleaseVersion}/i\ \ replaces: eclipse-che.v${lastPublishedPackageVersion}" \
     -e "s/${packageName}/eclipse-che/" \
     "${packageBaseFolderPath}/${lastPackagePreReleaseVersion}/che-operator.clusterserviceversion.yaml" \
-    > "${folderToUpdate}/${lastPackagePreReleaseVersion}/eclipse-che.v${lastPackagePreReleaseVersion}.clusterserviceversion.yaml"
+    > "${folderToUpdate}/${lastPackagePreReleaseVersion}/manifests/eclipse-che.v${lastPackagePreReleaseVersion}.clusterserviceversion.yaml"
 
     echo
+
     echo "   - Update the CRD files"
     cp "${packageBaseFolderPath}/${lastPackagePreReleaseVersion}/org_v1_che_crd.yaml" \
```

Comment:
Minor and up to you, but I think we can make this 'cp' command without new line delimiter `\`

Reply:
+1